analysis_scripts/analyze_doi_data/gender_with_bg_analysis.md

```diff
@@ -0,0 +1,150 @@
+springer_df = read_gender_files(springer_file)
+head(springer_df)
+```
+
+    ##     author                            doi year author_pos est_gender gender
+    ## 1      a s  doi:10.1007/s10586-017-1181-0 2017      first       MALE   MALE
+    ## 2       aa  doi:10.1007/s00059-013-3953-5 2013      first       MALE   MALE
+    ## 3       aa  doi:10.1007/s10751-012-0610-y 2012       last       MALE   MALE
+    ## 4    aadil  doi:10.1007/s12591-015-0244-z 2015      first       MALE   MALE
+    ## 5 aadithya  doi:10.1007/s12663-013-0500-0 2013       last       MALE   MALE
+    ## 6    aafke doi:10.1007/s00204-020-02953-6 2020      first       MALE   MALE
+
+``` r
+# read in the nature author data
+nature_file = file.path(proj_dir, "/data/author_data/nature_author_gender.tsv")
+nature_df = read_gender_files(nature_file)
+head(nature_df)
+```
+
+    ##   author                            doi year author_pos est_gender gender
+    ## 1 aakash doi:10.1038/s41586-020-03052-3 2020      first       MALE   MALE
+    ## 2  aaron        doi:10.1038/nature13124 2014      first       MALE   MALE
+    ## 3  aaron        doi:10.1038/nature25760 2018      first       MALE   MALE
+    ## 4  aaron  doi:10.1038/s41586-020-2944-y 2020      first       MALE   MALE
+    ## 5  aaron        doi:10.1038/nature13790 2014      first       MALE   MALE
+    ## 6  aaron        doi:10.1038/nature23912 2017      first       MALE   MALE
+
+### compare gender authorship proportions over all years
+
+Now lets look at all author publication gender and plot the trend over time. We see that the Springer corpus has many more articles sampled from it than the Nature corpus. We also see that there is an increase in Springer articles from 2005-2020; this is caused by name format changes. In this analysis, an article is only considered if it has a name where the gender is able to be guessed. In 2005, more journals were shortening the name such that only initials were used, thus reducing the number of articles able to be analyzed.
+
+Since the number of articles in Nature are small and the resulting proportion of male authors noisy, we will use the Springer background in the following comparison.
+
+<img src="gender_with_bg_analysis_files/figure-markdown_github/unnamed-chunk-4-1.png" width="50%" /><img src="gender_with_bg_analysis_files/figure-markdown_github/unnamed-chunk-4-2.png" width="50%" />
+
+## Compare Foreground and Background
+
+Now we will compare the identified quotes and cited authors from Nature News and compare the proportions of male speakers/authors against the previously shown background set of Springer articles.
+
+<img src="gender_with_bg_analysis_files/figure-markdown_github/unnamed-chunk-5-1.png" width="50%" /><img src="gender_with_bg_analysis_files/figure-markdown_github/unnamed-chunk-5-2.png" width="50%" /><img src="gender_with_bg_analysis_files/figure-markdown_github/unnamed-chunk-5-3.png" width="50%" />
```

Comment:
Why is there a discontinuity for the news-and-views line bar? Just curious as to whats happening with that plot.

Reply:
Ah good catch, it's because the ylim is to strict and the point wasn't caught. I will update it.

Reply:
https://github.com/greenelab/nature_news_disparities/pull/15/commits/cfcc85360a65f4d004832aca2d95173d09f7a786